.gitignore

```diff
@@ -1,24 +1,37 @@
+# OS artifacts
 .DS_Store
-*.gem
-*.rbc
-.bundle
-.config
+
+# IDEs
+.idea/
+
+# Documentation artifacts
 .yardoc
-InstalledFiles
 _yardoc
-coverage
 doc/
-lib/bundler/man
-pkg
-pkg/
 rdoc
+
+# Rubocop
+.rubocop-http*
+
+# test artifacts
 spec/reports
+coverage
 
-tmp
+# Appraisal
```

Comment:
fyi - This is the only addition, everything else is just organizing the ignores into meaningful chunks